src/components/likebox/LikeBox.jsx

```diff
@@ -1,17 +1,42 @@
 import React from "react";
+import { Link } from "react-router-dom";
+import { useLike } from "../../contexts/like-context";
+import { useWatchLater } from "../../contexts/watch-later-context";
+import Card from "../card/Card";
+import Card2 from "../card/Card2";
 
 export default function LikeBox() {
+  const { like } = useLike();
+  const likeId = like.like.map((item) => item._id);
+  const { watchLater } = useWatchLater();
+  const watchLaterId = watchLater.watchLater.map((item) => item._id);
+
   return (
     <div>
-      <div className="video-listing-container">
-        <div className="page-title">Liked video (0) </div>
-        <div className="page-video-container">
-          <div className="no-video-message">
-            There Is No Liked Video. Please Add{" "}
+      {like.like.length === 0 ? (
```

Comment:
Try to give more meaningful names for variables and lists. It will help in readability and debugging.
For ex: 

`likeState.likedList.length`